stock_prices/stock_prices.py

```diff
@@ -2,8 +2,26 @@
 
 import argparse
 
+# max profit variable start at inital index and compare 
+# min price make it index independent
+import math 
 def find_max_profit(prices):
-  pass
+  max_profit = -math.inf
+  min_price = math.inf
+  for price in prices:
```

Comment:
Awesome job this solution is very efficient. Maybe from now on try and comment out your approach (or use pen and paper) to try and accomplish a little more today.